service/service.py

```diff
@@ -277,6 +277,29 @@ def delete_recommendation_between_products(id, rel_id):
 
     return '', status.HTTP_204_NO_CONTENT
 
+######################################################################
+# DELETE ALL RELEATIONSHIP BY PRODUCT ID
+######################################################################
+@app.route('/recommendations/<int:id>', methods=['DELETE'])
+def delete_all_by_product_id(id):
```

Comment:
Great work Li, can you add in the description of this endpoint in the README file?

Reply:
Added.